src/LOAuth.ts

```diff
@@ -210,6 +208,7 @@ class LOAuth {
     storage.removeItem(storageKey);
   }
 
+  /* istanbul ignore next */
```

Comment:
branch coverage didn't get to 100% just for this line and I couldn't establish what was the cause as multiple tests hit this. I think the remaining coverage was because it wanted `options` to be undefined and there was no test coverage path that pumped in undefined for options

Reply:
`I think the remaining coverage was because it wanted options to be undefined and there was no test coverage path that pumped in undefined for options`: this signals that we might just want to change the signature to always require `options`.

Reply:
Maybe not related to this pr, but if we don't have a reason to use the defaults should we make the options required with no default?

Reply:
@aecorredor you're right, I remove the default and the coverage passes. I'll push that small change now